node/perf_hooks.ts

```diff
@@ -1,5 +1,44 @@
-export const { performance, PerformanceEntry } = globalThis as
-  & typeof globalThis
-  & {
+function unimplemented(functionName: string) {
+  throw new Error(
+    `Node.js performance method ${functionName} is not currently supported by JSPM core in the browser`,
```

Comment:
Deno has its own unimplemented function here which can be imported, see https://github.com/denoland/deno_std/blob/main/node/_utils.ts#L21.

Reply:
Ah, good catch 😅